axum-login/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "axum-login"
+version = "0.7.0"
+description = "🪪 User identification, authentication, and authorization for Axum."
+edition = "2021"
+homepage = "https://github.com/maxcountryman/axum-login"
+license = "MIT"
+keywords = ["authentication", "authorization", "login", "axum", "sessions"]
+categories = [
+    "web-programming::http-server",
+    "web-programming",
+    "database",
+    "asynchronous",
+]
+repository = "https://github.com/maxcountryman/axum-login"
+documentation = "https://docs.rs/axum-login"
+
+[dependencies]
+async-trait = "0.1.57"
+axum = { version = "0.6.20", default-features = false }
+http = "0.2.9"
+ring = "0.17.5"
+serde = "1"
+thiserror = "1.0.49"
+tower-cookies = "0.9.0"
+tower-layer = "0.3.2"
+tower-service = "0.3.2"
+tower-sessions = "0.4.1"
+tracing = { version = "0.1.40", features = ["log"] }
+urlencoding = "2.1.3"
+
+[dev-dependencies]
```

Comment:
This is mostly unneeded I believe?

Reply:
I've removed the ones we don't need for testing (doc-test specifically).